### PR TITLE
[REVIEW] NEXUS-5673: logspam in shadows

### PR DESCRIPTION
**Pull request by cstamas**
Related issue:
https://issues.sonatype.org/browse/NEXUS-5673

Fixed cases:
- original case from issue
- shadow put out of service
- checksum file deletion in M2 master

### CODE REVIEW

**Comment by ifedorenko** [comment]
+1 looks reasonable

File: nexus-core/src/main/java/org/sonatype/nexus/proxy/maven/LayoutConverterShadowRepository.java [hunk 1]
```diff
@@ -500,7 +500,19 @@ protected void deleteLink( final StorageItem item )
...
             request.getRequestContext().putAll( item.getItemContext() );
 
-            deleteItem( false, request );
+            try
+            {
+                deleteItem( false, request );
+            }
+            catch ( ItemNotFoundException e )
+            {
+                // just ignore it silently, this might happen when
```
Comment (kellyrob99) [inline nexus-core/src/main/java/org/sonatype/nexus/proxy/maven/LayoutConverterShadowRepository.java]: Please include the Jira for reference

**Comment by kellyrob99** [comment]
+1

**Comment by jdillon** [comment]
+1 why not

**Pull request merged**